### ISSUE TITLE
Propsal to add time stamp to Messages caused by Hercules 

### ISSUE THREAD

**Issue by fbi-ranger** [hercules-390/hyperion#145]
Messages generated by Hercules such as HHC00801I Processor CPXX: SIE Specifiacrion exception code XXXX does not show a date/time when issued to the log. This makes it difficult to match such an event with entries in a system log. It is proposed to enhance the message generation facility to issue an timestamp (Preferable YYYY-MM-DD-HH:MM:SS-ssssss) to the message log. 

Due to compatibility reasons this feature may controlled by the log command e.g. "log -t [newfile] name.

**Comment by Fish-Git** [comment]
I already developed code that neatly resolved this issue (refer to the "[JJ ZZ TODO item: timestamping messages](https://groups.yahoo.com/neo/groups/zHercules/conversations/messages/22214)" thread in the old Yahoo zHercules developers group) but have not had time to make the changes necessary to implement it the way I ultimately want to (i.e. as external library code like the way I believe we agreed Steve's SoftFloat changes are going to be implemented).

I'm currently juggling many, _many_ different things right now and haven't had time to work on it in a while now, but I'll see what I can do about maybe moving it up a position or two on my _looooong_ Hercules related "TODO" list.

I suppose I could go with my original non-external implementation which might be faster, but even that implementation still has some unresolved loose ends that still need tying up before it can be committed.

Bottom line: it's a known issue and I have a solution but it's not ready yet.

Much like a lot of what Enrico is working on I'm guessing.    `;-)`
